src/cluster.rs

```diff
@@ -489,11 +493,16 @@ impl ClusterConnection {
                         let kind = err.kind();
 
                         if kind == ErrorKind::Ask {
-                            asking = err.redirect_node().map(|x| x.0.to_string());
+                            redirected = err.redirect_node().map(|x| format!("redis://{}", x.0));
```

Comment:
It's the boy scout rule. I also fix the `ASK` redirect. Redis addresses should have the `"redis://"` prefix.